src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/feature_value.yml

```diff
@@ -0,0 +1,25 @@
+admin_feature_values_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureValueController::indexAction
+    _legacy_controller: AdminFeatures
+    _legacy_link: AdminFeatures
+    _legacy_feature_flag: feature
+  requirements:
+    featureId: \d+
+
+admin_feature_values_search:
+  path: /
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\CommonController::searchGridAction
```

Comment:
Oh so with the latest changes you managed to use the CommonController, that's great! Seems like the decision we made in the end was appropriate